tests/lib/outputs/outputs_test.py

```diff
@@ -201,6 +212,135 @@ def test_generate_common_csv_fields(self):
+            ],
+            "ENS-RD2022": ["mp.si.2.aws.s3.1"],
+            "NIST-CSF-1.1": ["ds_1"],
+            "RBI-Cyber-Security-Framework": ["annex_i_1_3"],
+            "FFIEC": ["d3-pc-am-b-12"],
+            "PCI-3.2.1": ["s3"],
+            "FedRamp-Moderate-Revision-4": ["sc-13", "sc-28"],
+            "FedRAMP-Low-Revision-4": ["sc-13"],
+        }
+        assert (
+            unroll_dict(test_compliance_dict)
+            == "CISA: your-systems-3, your-data-1, your-data-2 | CIS-1.4: 2.1.1 | CIS-1.5: 2.1.1 | GDPR: article_32 | AWS-Foundational-Security-Best-Practices: s3 | HIPAA: 164_308_a_1_ii_b, 164_308_a_4_ii_a, 164_312_a_2_iv, 164_312_c_1, 164_312_c_2, 164_312_e_2_ii | GxP-21-CFR-Part-11: 11.10-c, 11.30 | GxP-EU-Annex-11: 7.1-data-storage-damage-protection | NIST-800-171-Revision-2: 3_3_8, 3_5_10, 3_13_11, 3_13_16 | NIST-800-53-Revision-4: sc_28 | NIST-800-53-Revision-5: au_9_3, cm_6_a, cm_9_b, cp_9_d, cp_9_8, pm_11_b, sc_8_3, sc_8_4, sc_13_a, sc_16_1, sc_28_1, si_19_4 | ENS-RD2022: mp.si.2.aws.s3.1 | NIST-CSF-1.1: ds_1 | RBI-Cyber-Security-Framework: annex_i_1_3 | FFIEC: d3-pc-am-b-12 | PCI-3.2.1: s3 | FedRamp-Moderate-Revision-4: sc-13, sc-28 | FedRAMP-Low-Revision-4: sc-13"
+        )
+
+    def test_parse_html_string(self):
+        string = "CISA: your-systems-3, your-data-1, your-data-2 | CIS-1.4: 2.1.1 | CIS-1.5: 2.1.1 | GDPR: article_32 | AWS-Foundational-Security-Best-Practices: s3 | HIPAA: 164_308_a_1_ii_b, 164_308_a_4_ii_a, 164_312_a_2_iv, 164_312_c_1, 164_312_c_2, 164_312_e_2_ii | GxP-21-CFR-Part-11: 11.10-c, 11.30 | GxP-EU-Annex-11: 7.1-data-storage-damage-protection | NIST-800-171-Revision-2: 3_3_8, 3_5_10, 3_13_11, 3_13_16 | NIST-800-53-Revision-4: sc_28 | NIST-800-53-Revision-5: au_9_3, cm_6_a, cm_9_b, cp_9_d, cp_9_8, pm_11_b, sc_8_3, sc_8_4, sc_13_a, sc_16_1, sc_28_1, si_19_4 | ENS-RD2022: mp.si.2.aws.s3.1 | NIST-CSF-1.1: ds_1 | RBI-Cyber-Security-Framework: annex_i_1_3 | FFIEC: d3-pc-am-b-12 | PCI-3.2.1: s3 | FedRamp-Moderate-Revision-4: sc-13, sc-28 | FedRAMP-Low-Revision-4: sc-13"
+        assert (
+            parse_html_string(string)
+            == """
+➤CISA: your-systems-3, your-data-1, your-data-2
+
+➤CIS-1.4: 2.1.1
+
+➤CIS-1.5: 2.1.1
+
+➤GDPR: article_32
+
+➤AWS-Foundational-Security-Best-Practices: s3
+
+➤HIPAA: 164_308_a_1_ii_b, 164_308_a_4_ii_a, 164_312_a_2_iv, 164_312_c_1, 164_312_c_2, 164_312_e_2_ii
+
+➤GxP-21-CFR-Part-11: 11.10-c, 11.30
+
+➤GxP-EU-Annex-11: 7.1-data-storage-damage-protection
+
+➤NIST-800-171-Revision-2: 3_3_8, 3_5_10, 3_13_11, 3_13_16
+
+➤NIST-800-53-Revision-4: sc_28
+
+➤NIST-800-53-Revision-5: au_9_3, cm_6_a, cm_9_b, cp_9_d, cp_9_8, pm_11_b, sc_8_3, sc_8_4, sc_13_a, sc_16_1, sc_28_1, si_19_4
+
+➤ENS-RD2022: mp.si.2.aws.s3.1
+
+➤NIST-CSF-1.1: ds_1
+
+➤RBI-Cyber-Security-Framework: annex_i_1_3
+
+➤FFIEC: d3-pc-am-b-12
+
+➤PCI-3.2.1: s3
+
+➤FedRamp-Moderate-Revision-4: sc-13, sc-28
+
+➤FedRAMP-Low-Revision-4: sc-13
+"""
+        )
+
+    def test_parse_json_tags(self):
```

Comment:
Could you include some test to check the function's behaviour?

Reply:
Done